common-patterns/src/styles/slider.css

```diff
@@ -0,0 +1,14 @@
+@import url('./shared.css');
+
+.parent {
+  display: flex;
+  width: 40%;
+  align-items: center;
+  margin: 25px 0;
+}
+
+.child {
+  flex: 0 0 80%;
+  margin: 0 25px;
+  padding: 0 10px;
+}
```

Comment:
Nitpick: The way you currently have the CSS is best practices, but I think for this use case it could be nice to have the CSS in a `<style>` tag in the HTML.

It would make it easier for people to copy a specific pattern to their application. However, it would require some duplicate code, e.g. no `shared.css` file. What do you think? I'm good with either option, depending on what you think is better.

Reply:
🤔 Yeah that does make sense actually. I would personally appreciate the convenience so I think that's the move!